src/finufft.cpp

```diff
@@ -726,6 +725,7 @@ int FINUFFT_MAKEPLAN(int type, int dim, BIGINT *n_modes, int iflag, int ntrans,
       fprintf(stderr,
               "[%s] fwBatch would be bigger than MAX_NF, not attempting malloc!\n",
               __func__);
+      // FIXME: this error causes memory leaks. We should free phiHat1, phiHat2, phiHat3
```

Comment:
do the FIXME ;)

Reply:
Let's leave it in a separate PR. Otherwise in case of issues reverting this also removes an unrelated fix.